deployments/ansible/molecule/custom_vars/windows-verify.yml

```diff
@@ -45,23 +45,6 @@
       assert:
         that: not service_status.changed
 
-    - name: Download splunk-otel-collector 0.126.0 MSI
```

Comment:
Removed version check for the same reason as the [linux change](https://github.com/signalfx/splunk-otel-collector/pull/6284#discussion_r2127399483). We can revisit if anyone has a strong preference here.